d011-blackjack/Untitled-1.py

```diff
@@ -0,0 +1,128 @@
+#initially, randomly distribute two cards each to house and player 
+#if over 21, lose | if tie, house win | greater point wins otherwise
+#house has less than 16 points, add one card
+#player gets two cards, reveal
+#ask player if want another card
+#
+
+import random 
+card = ['A',
+         2,
+         3,
+         4,
+         5,
+         6,
+         7,
+         8,
+         9,
+        10,
+       'J',
+       'Q',
+       'K']
+    
+house_hand = []
+player_hand = []
+#random.shuffle(card)
```

Comment:
Let's remove commented out code from the file.

Reply:
Comment removed.